src/components/main/flight/ReceiptWithFlightScreen.js

```diff
@@ -1,232 +1,257 @@
+              Flight Number {""}
+              <Text style={{ fontFamily: "Poppins-bold" }}>
+                {flightChars} {flightNums}
+              </Text>
+            </Text>
+            <Text style={styles.header}>
+              The first step was taken. The rest is up to us!
+            </Text>
+            <Image source={leaf} style={styles.leaf_img} />
           </View>
-          <View style={styles.receiptContainer}>
-            <View style={styles.textRow}>
-              {/*Route*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('ROUTE')}</Text>
-              {/* Departure to Arrival */}
-              {tripIndex == 1 &&
-                <Text style={styles.receiptTextRight}>{depCityName} &#10230; {arrCityName}</Text>
-              }
-              {tripIndex == 2 &&
-                <Text style={styles.receiptTextRight}>{depCityName} &#10231; {arrCityName}</Text>
-              }
-            </View>
-            <Dash style={styles.dashedLine} dashColor={COLORS.lightgrey} dashGap={0} />
-            <View style={styles.textRow}>
-              {/*Carbon Footprints*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('CARBON FOOTPRINT (METRIC TONS)')}</Text>
-              <Text style={styles.receiptTextRight}>{footprint}</Text>
-            </View>
-            <Dash style={styles.dashedLine} dashColor={COLORS.lightgrey} dashGap={0} />
-            <View style={styles.textRow}>
-              {/*Total trees donated in transaction*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('TOTAL TREES')}</Text>
-              <Text style={styles.receiptTextRight}>{treeNum}</Text>
-            </View>
-            <Dash style={styles.dashedLine} dashColor={COLORS.lightgrey} dashGap={0} />
-            <View style={styles.textRow}>
-              {/*Years to neutralize*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('YEARS TO COMPENSATE')}</Text>
-              <Text style={styles.receiptTextRight}>{years}</Text>
-            </View>
-            <Dash style={styles.dashedLine} dashColor={COLORS.lightgrey} dashGap={0} />
-            <View style={styles.textRow}>
-              {/*Cost of transaction*/}
-              <Text style={styles.receiptTextLeft}>{i18n.t('PRICE')}</Text>
-              <Text style={styles.receiptTextRight}>${total_cost}</Text>
-            </View>
+          <View style={styles.bottomInnerView}>
+            <List>
+              {/* List Item 1 */}
+              <ListItem>
+                <Left>
+                  <Text style={styles.itemTitle}>Flight Distance (km)</Text>
+                </Left>
+                <Right>
+                  <Text style={styles.itemValue}>{distance}</Text>
+                </Right>
+              </ListItem>
+              {/* List Item 2 */}
+              <ListItem>
+                <Left>
+                  <Text style={styles.itemTitle}>
+                    Tons of CO
+                    <Text
```

Comment:
_Tons of CO2_ isn't showing up:
![image](https://user-images.githubusercontent.com/5898380/87211867-7f1fd480-c2d0-11ea-8ea1-f64562c96fd9.png)

HOWEVER, let's go back to showing **Carbon Footprint (Metric Tons)** as it's much more clear what it is compared to just Tons of CO2 and should fix the UI issue at the same time.